gslib/addlhelp/command_opts.py

```diff
@@ -149,8 +149,8 @@
               running gsutil from a cron job that logs its output to a file, for
               which the only information desired in the log is failures.
 
-  -u          Allows you to specify a user project to be billed for the request.
-              For example:
+  -u          Allows you to specify the ID or number of a user project to be\
```

Comment:
I think this is the only change line that says "user project" instead of just "project"

Reply:
My understanding is the project used with this flag is a little different than how projects are normally thought of: it's not the project in which the resource resides, but instead a project the requester ("user") is affiliated with. It's probably fine to ax "user", but I also see the use in being explicit about the nature of this particular project. I figured I'd punt and leave that particular text as is for the time being :P